tardis/model/base.py

```diff
@@ -84,6 +84,26 @@ def r_outer(self):
         return self.geometry.r_outer.values * self.geometry_units["r_outer"]
 
 
+class Geometry:
+    """
+    Holds information about model geometry
+
+    Parameters
+    ----------
+    radius : astropy.units.quantity.Quantity
```

Comment:
Give this the properties `r_inner`, `r_outer`, `v_inner`, and `v_outer` as numpy arrays containing units

Reply:
Make sure it has methods to compute volumes in each shell